src/nvim/getchar.c

```diff
@@ -3950,149 +3950,157 @@ void vim_unescape_csi(char_u *p)
   *d = NUL;
 }
 
-/*
- * Write map commands for the current mappings to an .exrc file.
- * Return FAIL on error, OK otherwise.
- */
-int 
-makemap (
+/// Write map commands for the current mappings to an .exrc file.
+/// Return FAIL on error, OK otherwise.
+int
+makemap(
     FILE *fd,
-    buf_T *buf           /* buffer for local mappings or NULL */
+    buf_T *buf  // buffer for local mappings or NULL
 )
 {
   mapblock_T  *mp;
-  char_u c1, c2, c3;
+  char_u      c1;
+  char_u      c2;
+  char_u      c3;
   char_u      *p;
   char        *cmd;
-  int abbr;
-  int hash;
-  bool did_cpo = false;
+  int         abbr;
+  int         hash;
+  bool        did_cpo = false;
```

Comment:
just for future reference, we generally don't align declarations. So this is reverting a previous un-align.